sql/mysqld.cc

```diff
@@ -533,6 +533,7 @@ ulong slave_retried_transactions;
 ulonglong slave_skipped_errors;
 ulong feature_files_opened_with_delayed_keys= 0, feature_check_constraint= 0;
 ulonglong denied_connections;
+ulong keepalive_time=0, keepalive_intvl=0, keepalive_probes=0;
```

Comment:
This appears to be a duplicate of sql_class.h definitions and not used. Same with mysqld.h